### PR TITLE
use "choose" instead of "select"

### PR DESCRIPTION
**Pull request by francescopellegrini**

### CODE REVIEW

File: src/clj/game/core/access.clj [hunk 1]
```diff
@@ -569,11 +569,11 @@
                             state {:base base :total (dec total)}
                             (conj already-accessed (:cid (first unrezzed))) args)
                           nil nil))
-              ;; more than one unrezzed upgrade. allow user to select with mouse.
+              ;; more than one unrezzed upgrade. allow user to choose with mouse.
```
Comment (NoahTheDuke) [inline src/clj/game/core/access.clj:572]: lol

File: src/clj/tasks/load_generator.clj [hunk 2]
```diff
@@ -113,7 +113,7 @@
                                      :client-id runner-client-id
                                      :?data {:gameid game-id
                                              :password ""}})
-      ;; Select decks
+      ;; Choose decks
```
Comment (NoahTheDuke) [inline src/clj/tasks/load_generator.clj:116]: lol overeager, but this isn't hurting anything

**Comment by NoahTheDuke** [comment]
I think if you undo the changes to everything that's not related to the user clicking a card while in a game ("choose deck", "choose round"), this is good to go

**Comment by francescopellegrini** [comment]
> I think if you undo the changes to everything that's not related to the user clicking a card while in a game ("choose deck", "choose round"), this is good to go

Should be ok now :)

**Pull request merged**